ci/docs-build-verification.sh

```diff
@@ -33,7 +33,8 @@ TMP_DIR=$(mktemp -d)
 
 # Go to a temporary directory and simulate the merge
 cd ${TMP_DIR}
-git clone ${CLONE_URL}
+mkdir -p unity-gdk
```

Comment:
actually I don't think this line is needed

Reply:
+1